recipes/output_viewer/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "output_viewer" %}
+{% set version = "1.2.5" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/esgf/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0ce542616a089b1069b474e0b6c2f4373b5c104670351d81b1750f85951c7959
+
+build:
+    number: 0
+    noarch: python
+    script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python {{ python }}
+    - setuptools
+    - requests
+    - pip
+  run:
+    - python {{ python }}
+    - requests
+
+test:
+   imports:
+       - output_viewer
+
+about:
+  home: https://github.com/esgf/output_viewer
+  summary: The Output Viewer is designed to provide a framework for viewing arbitrary output from diagnostics scripts, metrics, or any program that produces a ton of different files that you want to look at it in an easy-to-use fashion.
+  license: ESGF
+  license_file: LICENSE
+
+extra:
+    recipe-maintainers:
+        - doutriaux1
+        - zshaheen
```

Comment:
@zshaheen are you okay with being maintainer on this feedstock?

Reply:
@mariusvniekerk thanks for asking. @zshaheen and I work together he is actually the one that uses this the most as part of his cdp package, which depnds on this and is coming down as a staged recipe as soon as this is a feedstock.

Reply:
@mariusvniekerk by the way when you try to add a few packages with chained dependencies, is there a way to put them all in one staged-recipe? Or do we have to go through the process 3 times?

Reply:
@mariusvniekerk Yes, I'm fine with this!